mountpoint-s3-client/tests/common/mod.rs

```diff
@@ -52,6 +58,32 @@ pub async fn get_test_sdk_client() -> s3::Client {
     s3::Client::new(&config)
 }
 
+pub fn get_test_access_point_alias_and_prefix(test_name: &str, access_point_type: AccessPointType) -> (String, String) {
+    let access_point = match access_point_type {
+        AccessPointType::SingleRegion => {
+            std::env::var("S3_ACCESS_POINT_ALIAS").expect("Set S3_ACCESS_POINT_ALIAS to run integration tests")
+        }
+        AccessPointType::ObjectLambda => {
+            std::env::var("S3_OLAP_ALIAS").expect("Set S3_OLAP_ALIAS to run integration tests")
+        }
+        AccessPointType::MultiRegion => {
+            std::env::var("S3_MRAP_ALIAS").expect("Set S3_MRAP_ALIAS to run integration tests")
+        }
+    };
+
+    // Generate a random nonce to make sure this prefix is truly unique
+    let nonce = OsRng.next_u64();
+
+    // Prefix always has a trailing "/" to keep meaning in sync with the S3 API.
+    let prefix =
+        std::env::var("S3_BUCKET_TEST_PREFIX").expect("Set S3_ACCESS_POINT_TEST_PREFIX to run integration tests");
```

Comment:
`S3_BUCKET_TEST_PREFIX` or `S3_ACCESS_POINT_TEST_PREFIX`?

Reply:
Depending on whether we just want `S3_BUCKET_TEST_PREFIX` here, it may be worth to just split out the prefix part from `get_test_bucket_and_prefix` and use it everywhere.